.github/workflows/ci_pipeline.yml

```
@@ -135,46 +135,105 @@ jobs:
       with:
-        name: coverage-ai-${{ matrix.os }}
-        path: dist/coverage-ai*
+        name: cover-agent-${{ matrix.os }}
+        path: dist/cover-agent*
+
+  e2e-test:
+    # This job will run after building binary because it needs the binary to run the E2E tests in Docker
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)